src/Chainweb/CutDB.hs

```diff
@@ -431,45 +458,49 @@ startCutDb config logfun headerStore payloadStore cutHashesStore = mask_ $ do
     -- 3. exitence of dependencies
     -- 4. full validation
     --
-    readInitialCut = withTableIter (_getRocksDbCas cutHashesStore) $ \it -> do
-        tableIterLast it
-        go it
-      where
-        -- TODO: should we limit the search to a certain number of attempts
-        -- or iterate in increasinly larger steps?
-        go it = tableIterValue it >>= \case
-            Nothing -> do
-                logg Info "using initial cut from cut db configuration"
-                return $! _cutDbParamsInitialCut config
```

Comment:
This is replaced with `view cutMap $ genesisCut v` below. Is `_cutDbParamsInitialCut` still used elsewhere or can it be removed?

Reply:
Erm it's not used elsewhere. It should be used for the "load initial cut from file" behavior, but we don't have that right now.

Reply:
Actually `_cutDbParamsInitialCut` *was* removed in this PR